ILMerge/AssemblyInfo.cs

```diff
@@ -1,57 +0,0 @@
-using System.Reflection;
-using System.Runtime.CompilerServices;
-
-//
-// General Information about an assembly is controlled through the following 
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-//
-[assembly: AssemblyTitle("ILMerge")]
-[assembly: AssemblyDescription("A tool for merging multiple .NET assemblies into a single assembly.")]
```

Comment:
FYI - the contents of this file are automatically generated on-build by VS 2017.